modules/core/src/lib/api/register-loaders.js

```diff
@@ -1,24 +1,31 @@
+import {global} from '@loaders.gl/loader-utils';
 import {normalizeLoader} from '../loader-utils/normalize-loader';
 
-let registeredLoaders = [];
+// Store global registered loaders on the global object to increase chances of cross loaders-version interoperability
+// NOTE: This use case is not reliable but can help when testing new versions of loaders.gl with existing frameworks
+global.loaders = global.loaders || {};
```

Comment:
Orphan code is problematic with aggressive bundler/optimizers. I recommend moving this into `getRegisteredLoaders`